.drone.yml

```diff
@@ -0,0 +1,59 @@
+---
+kind: pipeline
+name: amd64
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: build
+  image: rancher/dapper:v0.4.1
+  commands:
+  - dapper ci
+  volumes:
+  - name: docker
+    path: /var/run/docker.sock
+
+- name: publish-master
+  image: plugins/gcs
+  settings:
+    acl:
+      - allUsers:READER
+    cache_control: "public,no-cache,proxy-revalidate"
+    source: dist/artifacts
+    target: releases.rancher.com/harvester-node-driver/${DRONE_BRANCH}
+    token:
+      from_secret: google_auth_key
+  when:
+    ref:
+      include:
+        - "refs/heads/master"
+        - "refs/heads/release/v*"
+    event:
+      - push
```

Comment:
I think this is redundant, we may only need one for both master and tag push/release.